doc/api/stream.md

```diff
@@ -1801,10 +1801,10 @@ This function MUST NOT be called by application code directly. It should be
 implemented by child classes, and called by the internal `Writable` class
 methods only.
 
-The `writable._writev()` method may be implemented in addition to
-`writable._write()` in stream implementations that are capable of processing
-multiple chunks of data at once. If implemented, the method will be called with
-all chunks of data currently buffered in the write queue.
+The `writable._writev()` method may be implemented in addition (but not
+instead of) to `writable._write()` in stream implementations that are capable
```

Comment:
Not sure the addition of `(but not instead of)` adds anything significant here.

Reply:
Would have helped me. I thought it was enough to implement `writev` (which is probably something that we could make true) before I noticed it didn't work.

Reply:
I'm OK with adding the parenthetical if it helps people (and I imagine it would). There's a preposition that should be moved, I think. Instead of "in addition (but not instead of) to", it sounds more natural (and hopefully is easier to understand too?) if it's "in addition to (but not instead of)". In other words, move "to" from after the parenthetical to before it.